eng/Directory.Build.Common.props

```diff
@@ -243,6 +243,14 @@
     <RunSettingsFilePath Condition="'$(AZURE_SKIP_DEFAULT_RUN_SETTINGS)' != 'true'">$(RepoEngPath)\nunit.runsettings</RunSettingsFilePath>
   </PropertyGroup>
 
+  <!-- Set global test timeout of 5 seconds -->
+  <ItemGroup Condition="'$(IsTestProject)' == 'true'">
```

Comment:
How do we override? Also what about live tests?

Reply:
The most local attribute takes precedence. So to override, a specific class, or test could be marked with a longer timeout.

For live tests, I'll look into if we can possibly set this dynamically based on test mode.